db/migrate/20201011132400_create_observation_views.rb

```diff
@@ -0,0 +1,13 @@
+class CreateObservationViews < ActiveRecord::Migration[5.2]
+  def up
+    create_table :observation_views, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
+      t.integer  :observation_id
+      t.integer  :user_id
+      t.datetime :last_view
+    end
+  end
+
+  def down
+    drop_table :observation_views
+  end
```

Comment:
This is a little shorter:
```suggestion
  def change
    create_table :observation_views, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
      t.integer  :observation_id
      t.integer  :user_id
      t.datetime :last_view
    end
  end
```

Reply:
Thanks, Joe!  I wanted to do this, but after an hour of trying to access the docs for the correct syntax, I finally gave up and went with the old style. :)